src/reactable.jsx

```diff
@@ -344,6 +344,7 @@
             var Ths = [];
             for (var index = 0; index < this.props.columns.length; index++) {
                 var column = this.props.columns[index];
+                var thClass = column.key
```

Comment:
Can we namespace this? I feel like just naming it after the column key indiscriminately could result in unexpected clashes. Maybe `'reactable-th-' + column.key`?